zad6/app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+class Category < ApplicationRecord
+	has_many :product_categories
+	has_many :products, :through => :product_categories
```

Comment:
`:through => :product_categories` to stary zapis hasha i jest tym samym co nowe `through: :product_categories`